nixos/modules/services/torrent/transmission.nix

```diff
@@ -358,7 +356,7 @@ in
           "transmission/${downloadsDir}"
           "transmission/${watchDir}"
         ];
-        StateDirectoryMode = mkDefault 750;
+        StateDirectoryMode = cfg.downloadDirPermissions;
```

Comment:
Please keep 750 and let users add themselves to the transmission group.

Reply:
How do I add a user to the multiple groups declaratively though?